rust/operator-binary/src/main.rs

```diff
@@ -34,6 +34,24 @@ pub mod built_info {
     pub const CARGO_PKG_VERSION: &str = env!("CARGO_PKG_VERSION");
 }
 
+#[derive(Debug, Snafu)]
```

Comment:
Cannot comment on the code line but above should be 
```
pub mod built_info {
    include!(concat!(env!("OUT_DIR"), "/built.rs"));
}
```
and adapt e.g. `CARGO_PKG_VERSION` to `PKG_VERSION` (see main file here https://github.com/stackabletech/listener-operator/pull/183/files)